client/src/theme/Components/_toggle_primary.scss

```diff
@@ -0,0 +1,40 @@
+@mixin togglePrimary($height) {
+  position: relative;
+  height: $height;
```

Comment:
I don't think we need a fixed height here unless one day there are multiple toggles next to eachother that need to be the same height.